errors/go.mod

```diff
@@ -4,15 +4,25 @@ go 1.18
 
 require (
 	github.com/pkg/errors v0.9.1
-	github.com/stretchr/testify v1.7.0
-	google.golang.org/grpc v1.45.0
+	github.com/stretchr/testify v1.7.1
+	github.com/tendermint/tendermint v0.35.6
```

Comment:
We may not want to have a tendermint dependency in an error package, but these helpers function seems to belong more here.